docs/oneliners.rst

```diff
@@ -0,0 +1,347 @@
+.. -*- mode: rst -*-
+
+Oneliners
+=========
+
+Yellowbrick's quick methods are visualizers in a single line of code!
+
+Yellowbrick is designed to give you as much control as you would like over the plots you create, offering parameters to help you customize everything from color, size, and title to preferred evaluation or correlation measure, optional bestfit lines or histograms, and cross validation techniques. To learn more about how to customize your visualizations using those parameters, check out the :doc:`api/index`.
+
+But... sometimes you just want to build a plot with a single line of code!
+
+On this page we'll explore the Yellowbrick quick methods (aka "oneliners"), which return a fully fitted, finalized visualizer object in only a single line.
+
+.. note:: This page illustrates oneliners for some of our most popular visualizers for feature analysis, classification, regression, clustering, and target evaluation, but is not a comprehensive list. Nearly every Yellowbrick visualizer has an associated quick method!
```

Comment:
Perhaps we should make documenting the quick methods a priority for the Fall semester.